src/ipc.rs

```diff
@@ -200,10 +213,13 @@ impl<T> IpcReceiver<T> where T: for<'de> Deserialize<'de> + Serialize {
     }
 
     /// Non-blocking receive
-    pub fn try_recv(&self) -> Result<T, bincode::Error> {
+    pub fn try_recv(&self) -> Result<T, TryRecvError> {
```

Comment:
While we're at it, should we change the API of `recv`? We'll probably want to at some point, and it saves another breaking change.